examples/mobile/ScheduleApp/css/style.css

```diff
@@ -0,0 +1,24 @@
+html,
+body {
+    width: 100%;
+    height: 100%;
+    margin: 0 auto;
+    padding: 0;
+    background-color: #222222;
+    color: #ffffff;
+}
+.page {
+    width: 100%;
+    height: 100%;
+    display: table;
+}
+.contents {
+    display: table-cell;
+    vertical-align: middle;
+    text-align: center;
+    -webkit-tap-highlight-color: transparent;
+}
+#content-text {
```

Comment:
Is it right "#"? Is it your intended?

Reply:
This CSS file was generated by Tizen Studio, I will remove it

Reply:
I removed entire file.